ci-operator/config/openshift/sig-storage-local-static-provisioner/openshift-sig-storage-local-static-provisioner-release-4.3.yaml

```diff
@@ -0,0 +1,39 @@
+base_images:
```

Comment:
Do you actually have all these branches that you actually want to test the same way?